forem/forem
# Bump imgproxy to 2.0

## What type of PR is this? (check all applicable)

## Description
Imgproxy 2.0 has some naming changes that require a manual upgrades.
## Related Tickets & Documents
n/a
## QA Instructions, Screenshots, Recordings
This is a dependency update. To verify that imgproxy is functioning normally, please see our [imgproxy setup guide](https://docs.forem.com/installation/imgproxy/)
## Added tests?
- [x] no because I don't think it's necessary

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
n/a
